exercises/5th_task/main.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+type Node struct {
+	Value int
+	Next  *Node
+}
+
+func (n Node) asSlice() (slice []int) {
+	i := 0
+	for n.Next != nil && i < 10 {
```

Comment:
Nit: Why 'i < 10'? You could simply collect the entire list into the slice

Reply:
it was added for debug purpeses and I forgot to remove it.